Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /root
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-    apt-get install -y curl gawk groovy nodejs openjdk-8-jre-headless php python ruby
+    apt-get install -y curl gawk groovy nodejs openjdk-8-jre-headless php python ruby monodevelop
```

Comment:
Would be great to keep these in alphabetical order.

Primarily because it helps to prevent merge conflicts when separate branches both add things at the end.

Reply:
Good catch, fixed!